dash/development/_r_components_generation.py

```diff
@@ -129,22 +136,34 @@
 \\docType{{package}}
 \\name{{{package_name}-package}}
 \\alias{{{package_name}}}
-\\title{{{pkg_help_header}}}
+\\title{{{pkg_help_title}}}
 \\description{{
-{pkg_help_desc}
-}}
-\\seealso{{
-Useful links:
-\\itemize{{
-  \\item \\url{{https://github.com/plotly/{lib_name}}}
-  \\item Report bugs at \\url{{https://github.com/plotly/{lib_name}/issues}}
-}}
+{pkg_help_description}
 }}
 \\author{{
-\\strong{{Maintainer}}: {package_author}
+\\strong{{Maintainer}}: {maintainer}
 }}
 """
 
+component_helpers = """
+dash_assert_valid_wildcards <- function (attrib = list("data", "aria"), ...)
+{
+    args <- list(...)
+    validation_results <- lapply(names(args), function(x) {
+        grepl(paste0("^", attrib, "-[a-zA-Z0-9]{1,}$", collapse = "|"),
```

Comment:
We want to accept `-` and `_` too (and maybe others, but let's start there) https://stackoverflow.com/questions/25897910/valid-characters-in-custom-data-attribute-name-in-html5

Also a little bit shorter regex (though longer code to create it 😄):

```suggestion
        grepl(paste0("^(", paste0(attrib, collapse="|"), ")-[a-zA-Z0-9_-]+$"),
```